tests/licensedcode/data/additional_licenses/additional_license_combined_test.expected.json

```diff
@@ -386,9 +388,6 @@
       "license_detections": [
         {
           "license_expression": "example-installed-1 AND example-installed-2 AND example1 AND example2 AND apache-2.0",
-          "detection_log": [
-            "not-combined"
-          ],
```

Comment:
Even if empty, "detection_log" should be there always.

Reply:
Refer https://github.com/nexB/scancode-toolkit/issues/3266#issuecomment-1443672136, we have moved detection log to only be there with `--license-text-diagnostics` as we don't want this always.

Reply:
From @pombredanne : we might want to have both, as there could be two different use cases here:
1. having usable matched_text for generating notices, with the detection_diagnostics available.
2. having matched text diagnostics to see what words were matched.

Reply:
Done, we have a follow-up at https://github.com/nexB/scancode-toolkit/issues/3288